bin/tictactoe

```diff
@@ -1,3 +1,25 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+puts "Welcome to Tic Tac Toe!"
+puts "press 0 for a 0 players game"
+puts "press 1 for a 1 players game"
+puts "press 2 for a 2 players game"
+input = gets.strip
+if input == "0"
+	game = Game.new(player_1 = Player::Computer.new("X"), player_2 = Player::Computer.new("O"), board = Board.new)
```

Comment:
Being consistent with your whitespace, particularly using 2 spaces for indentation, will make your code more readable.